### PR TITLE
Updated the vsvip object with tenant, cloud and vrfcontext ref in sdk-module examples

### PR DESCRIPTION
**Pull request by pshrikant456**

### CODE REVIEW

File: ansible/avisdk_modules_examples/complex_app/complex_app.yml [hunk 1]
```diff
@@ -415,4 +417,4 @@
         cloud_ref: "/api/cloud/?name=Test-Cloud"
         se_group_ref: '/api/serviceenginegroup/?name=sample_serviceenginegroup'
         vrf_context_ref: "/api/vrfcontext/?name=sample_vrfcontext"
-        vsvip_ref: /api/vsvip/?name=vsvip-{{ app_name_2 }}-Default-Cloud
+        vsvip_ref: /api/vsvip/?name=vsvip-{{ app_name_2 }}
```
Comment (chaitanyaavi) [inline ansible/avisdk_modules_examples/complex_app/complex_app.yml:420]: New line missing

File: ansible/avisdk_modules_examples/complex_app/complex_app.yml [hunk 2]
```diff
@@ -36,12 +36,12 @@
...
-          username: "root"
-          datacenter: "10GTest"
-          management_network: "/api/vimgrnwruntime?name=Mgmt_Arista_3"
+          username: "username"
+          datacenter: "wdc-02-vc20"
+          management_network: "/api/vimgrnwruntime/?name=vxw-dvs-34-virtualwire-3-sid-2200002-wdc-02-vc20-avi-mgmt"
           privilege: "WRITE_ACCESS"
-          vcenter_url: "10.10.2.10"
-          password: vmware
+          vcenter_url: "10.79.22.40"
```
Comment (chaitanyaavi) [inline ansible/avisdk_modules_examples/complex_app/complex_app.yml:43]: Use all these Vcenter inputs as vars

**Comment by sabandi** [comment]
overall LGTM. Please fix the issues mentioned in comments.

File: ansible/avisdk_modules_examples/complex_app/complex_app.yml (hunk 1 above)
Comment (pshrikant456) [reply ansible/avisdk_modules_examples/complex_app/complex_app.yml]: Updated the file with adding new line.

File: ansible/avisdk_modules_examples/complex_app/complex_app.yml (hunk 2 above)
Comment (pshrikant456) [reply ansible/avisdk_modules_examples/complex_app/complex_app.yml]: Updated the playbook with getting the Vcenter input from creds file.

**Review by sabandi — APPROVED** [review]
@patilshrikant786 please merge if we don't have any pending items.

**Comment by pshrikant456** [comment]
> @patilshrikant786 please merge if we don't have any pending items.

for the sdk module example issue all changes are done, but all devops examples are not tested yet so if we found any issue in that then i will create a another PR

**Pull request merged**